node/core/dispute-coordinator/src/participation/queues/mod.rs

```diff
@@ -277,8 +247,9 @@ struct CandidateComparator {
 	/// just using the lowest `BlockNumber` of all available including blocks - the problem is,
 	/// that is not stable. If a new fork appears after the fact, we would start ordering the same
 	/// candidate differently, which would result in the same candidate getting queued twice.
-	relay_parent_block_number: BlockNumber,
+	relay_parent_block_number: Option<BlockNumber>,
 	/// By adding the `CandidateHash`, we can guarantee a unique ordering across candidates.
+	/// Additionally if `BlockNumber` can't be obtained the `CandidateHash` is used for ordering.
```

Comment:
I think we also use CandidateHash for ordering if BlockNumber is the same between two disputes.

Reply:
The whole sentence is misplaced. I'll rework this paragraph.